.github/ISSUE_TEMPLATE/PLIP.md

```diff
@@ -3,6 +3,7 @@ name: "\U0001F680 PLIP"
 about: Plone Improvement Proposal
```

Comment:
To align with Volto and have it better defined, which we found was necessary to direct first-timers toward bug reports or feature requests. We also used enumeration on the file names so that they would sort in an order that would put PLIPs after bug reports and feature requests, making it a less likely choice. See https://github.com/plone/volto/tree/main/.github/ISSUE_TEMPLATE.

```suggestion
about: 'A Plone Improvement Proposal (PLIP) is a change to a Plone package that would affect everyone who uses that package.'
```

Reply:
Yeah, I wasn't happy about the current `about`, and you pointed it out.

Other sources should align with the authoritative source, which is the Plone 6 Documentation [PLIPs page](https://6.docs.plone.org/contributing/core/plips.html).

I'd say let's keep it brief, and refer to the authoritative source. The glossary isn't authoritative, but it's a brief summary. I've edited my suggestion. Please take another look.